src/applications/gqrx/receiver.cpp

```diff
@@ -206,7 +217,13 @@ void receiver::set_input_device(const std::string device)
         tb->disconnect(src, 0, iq_swap, 0);
     }
 
-    src.reset();
+    //temporarily connect dummy source to ensure that previous device is closed
```

Comment:
Thanks for the helpful comment explaining the purpose of the dummy device connect 👍